--- a/lib/anytone_interface.cc
+++ b/lib/anytone_interface.cc
@@ -318,24 +318,26 @@

AnytoneGD32Interface::identifier(const ErrorStack &err) {
  if (! _info.isValid())
    return RadioInfo();
  if ("D868UVE" == _info.name) {
    return RadioInfo::byID(RadioInfo::D868UVE);
  } else if ("D6X2UV" == _info.name) {
    return RadioInfo::byID(RadioInfo::DMR6X2UV);
  } else if ("D6X2UV2" == _info.name) {
    return RadioInfo::byID(RadioInfo::DMR6X2UV2);
  } else if ("D878UV" == _info.name) {
    return RadioInfo::byID(RadioInfo::D878UV);
  } else if ("D878UV2" == _info.name) {
    return RadioInfo::byID(RadioInfo::D878UVII);
+ } else if ("D168UV" == _info.name) {
+   return RadioInfo::byID(RadioInfo::D878UVII);
  } else if ("D578UV" == _info.name) {
    return RadioInfo::byID(RadioInfo::D578UV);
  }  else if ("D578UV2" == _info.name) {
    return RadioInfo::byID(RadioInfo::D578UVII);
  }

  errMsg(err) << tr("Unsupported AnyTone radio '%1', HW rev. '%2'.")
                 .arg(_info.name).arg(_info.version);
  return RadioInfo();
}

